workflows/wf3/1-fetch-metadata.js

```diff
@@ -38,10 +24,31 @@ fn(state => {
   return state;
 });
 
+get(
+  `${$.workbookBase}/worksheets('omrs-form-metadata')/usedRange`,
+  {},
+  state => {
+    const [headers, ...rows] = state.data.values.slice(0);
```

Comment:
Why the slice here?

Reply:
To separate the header row from other rows with content

Reply:
But why not ` const [headers, ...rows] = state.data.values` ?

Reply:
removed `.slice(0)`